.github/workflows/validate-rules-syntax.yaml

```diff
@@ -0,0 +1,100 @@
+on: pull_request
+permissions:
+  contents: read
+  pull-requests: read
+jobs:
+  validate_loki_recording_rules:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Create tmp dir 
+        run: mkdir -p validation/recording-rules
+
+      - name: Generate Loki recording rules
+        run: yq e '.spec' loki/recording-rules.yaml > validation/recording-rules/rules.yaml
```

Comment:
What `yq` invocation is this?
Generally at BitGo we assume yq is https://github.com/kislyuk/yq, rather than http://mikefarah.github.io/yq/

Reply:
It took my some time to figure out that the `yq` included in Ubuntu is coming from http://mikefarah.github.io/yq/ so it has a slightly different syntax.

I'd favor using a different `yq` implementation which comes already with Ubuntu rather than installing another package for simplicity.